js/hoverzoom.js

```diff
@@ -635,9 +635,9 @@ var hoverZoom = {
             return (contentOverflows && overflowShown) || (alwaysShowScroll);
         }
 
-        const videoExtensions = new Set(['3gpp', 'm4v', 'mkv', 'mp4', 'ogv', 'webm']);
+        const videoExtensions = new Set(['3gpp', 'm4v', 'mkv', 'mp4', 'ogv', 'webm', 'm4s']);
         const videoExtensionsWithGif = new Set(['3gpp', 'gif', 'gifv', 'm4v', 'mkv', 'mp4', 'ogv', 'webm']);
-        const audioExtensions = new Set(['flac', 'm4a', 'mp3', 'oga', 'ogg', 'opus', 'wav']);
+        const audioExtensions = new Set(['flac', 'm4a', 'mp3', 'oga', 'ogg', 'opus', 'wav', 'm4s']);
```

Comment:
@GrosPoulet How can `m4s` be both video and audio at the same time? None of existing extensions are shared between video and audio. How is having the same extension going to work?

Reply:
Bilibili (and other websites) streams content using little M4S chunks of data: some audio chunks + some video chunks.
video.js player handles both perfectly:

![bilibili](https://github.com/user-attachments/assets/d7e199ee-5209-4a41-9fc6-936b7a5d8032)